src/services/kubernetes/per-deployment/new-client.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
```

Comment:
Not super consequential for a UI, but would it make sense to tie this dynamically to a package.json version number?  Or a concatenation of package.json version and git hash?  @josepot

Reply:
This refers to the kubernetes api version, nothing to do with our apps.

Reply:
The BUILD_VERSION you see a few line below is where we inject our version number. In the case of ephemeral PR deployments, this will be a hash calculated from the file tree. In permanent deployments, it uses git-describe (and therefore a derivation of git tags), which tracks the version of the project as a whole.